style.css

```diff
@@ -37,19 +37,93 @@ body {
 @media screen and (min-width: 375px) {
   /* header section */
   .header {
-    display: flex;
     height: 5vh;
-    justify-content: center;
+    position: relative;
+    z-index: 9999;
   }
 
   .header-container {
-    display: flex;
-    justify-content: end;
-    width: 90%;
+    width: 100%;
+    position: relative;
   }
 
+  .header button {
+    position: absolute;
+    z-index: 1;
+    right: 10%;
+    top: 15px;
+    background-color: transparent;
+    border: none;
+  }
+
+  /* .mobile-menu-toggle:hover nav {
+    transform: translateY(0);
+  } */
+
   .header nav {
-    display: none;
+    /* display: none; */
+    display: block;
+    position: absolute;
+    height: 100vh;
+    transform: translateY(-100%);
+    transition: transform 450ms ease-in 100ms, background-color 400ms ease-in;
+  }
+
+  .header nav[data-visible='true'] {
+    transform: translateY(0%);
+    transition: transform 450ms ease-in, background-color 200ms ease-in 450ms;
+    background-color: #e8e9f0;
+  }
+
+  .header nav ul {
+    box-sizing: border-box;
+    padding-top: 64px;
+    margin: 0;
+    background-color: white;
+    width: 100vw;
+    height: 92vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    gap: 40px;
+    list-style: none;
+    border-bottom-left-radius: 40px;
+    border-bottom-right-radius: 40px;
+  }
```

Comment:
- [ ] If we take a look at the Figma design you will notice that the menu container takes up the full-screen size.  💪🏼 so kindly ensure that you follow the Figma design system as much as possible, by minimizing the bottom grey space 👍🏼 

### Your design
![image](https://github.com/ichsansandy/portfolio-microverse/assets/68030297/283ba22f-c37d-4cc1-846e-50cb63ac20b3)

### Figma Design
![image](https://github.com/ichsansandy/portfolio-microverse/assets/68030297/86391657-e4d8-4543-8a2c-75a81f7e3ab4)